router/config.go

```diff
@@ -314,6 +317,29 @@ func (c *UpdatableAddress) Store(address transport.Address) {
 	c.wrapped.Store(address)
 }
 
```

Comment:
Marshall/UnmarshallYaml should have pointer receivers for consistency with the other methods